manifest.json

```
@@ -148,6 +148,13 @@
       "id": "dragdropselected",
       "mod_version": "3"
     },
+    {
+      "description": "Adds a popup that displays the curve of Penner-styled easing functions.",
+      "version": "1.0",
+      "remote": "https://github.com/ThaCuber/lite-xl-easingpreview",
```

You must specify a commit hash like this:
```suggestion
      "remote": "https://github.com/ThaCuber/lite-xl-easingpreview:ea349d12a9570264408dc1567af660b8d4568d3d",
```

Might also be worth it to add a `name` field.

---

whi though (the commit hash)

---

To pin the version to the commit (and as a way to avoid malicious changes).

---

also, can I write "Easing Previewer" for the `name` field?

---

Yep.